### ISSUE TITLE
Ignore app menu?

### ISSUE THREAD

**Issue by Naomitr0n** [jd1378/otphelper#74]
I've had some false positives on Google Keep where it grabs the code `600`. Here's the two ignore entries from it so far, but there have been others I've missed too:
```
app:com.google.android.keep:nid:4
app:com.google.android.keep:nid:3
```

Best I can figure is it's pulling from a "work in progress" note on my project car every time it's an undismissed notification and another reminder is shown:

> Causes stalls from Park to Reverse -- idle is WAY too high, should be 600 to 800

While pattern matching improvements might fix that (and it might even be something else, I'm really not sure), what I'm curious about is if there's a way to show a list of installed apps, and have the user manually select apps to ignore outright that then get added to the list as a preventative fix, instead of only fixing it as it happens. F-Droid for example has had `file` detected a few times but I've never caught it in time to ignore it (I'm not sure why it goes away either).

Obviously to reduce clutter you could suggest to the user to only select apps that might send notifications as those that don't won't be detected anyway.

Also desired fix:
[Moved to new issue]

**Comment by jd1378** [comment]
Hi
This should be its own issue
And I think what you say makes sense
The only problem is me
It would be great if someone could help improve the app by contributing
Otherwise it may a long time for me to implement features

**Comment by Naomitr0n** [comment]
Split out the last part to #75

**Comment by jd1378** [comment]
I have added a History screen where it shows which app triggered the detection, along with it's notification Tag and ID
it should be more clear now
it will be released on 1.16.0
a release candidate is available, please test and provide feedback if you can
https://github.com/jd1378/otphelper/releases/tag/v1.16.0-rc
